webui.py

```diff
@@ -1094,7 +1093,7 @@ def run_simple(self):
 
         if path == '/':
             html = [
-                '<html><head><title>Simple Index</title><meta name="api-version" value="2" /></head>',
+                '<!DOCTYPE html><html><head><title>Simple Index</title></head>',
```

Comment:
The !DOCTYPE addition is sound, however I'd like to stage it in a separate PR if possible.